### PR TITLE
Integrate API and some other stuff

### PR DESCRIPTION
**Pull request by lankybox02**

### CODE REVIEW

**Comment by Turtle84375** [comment]
This is a huge set of changes, and it's also where I need to make the big decision to either use what you've come up with (Node.js) or continue developing the current auth (Python), which works great as of right now.

I'll set up a preview deployment with Vercel to see how I like it before I push it through.

**Comment by lankybox02** [comment]
@Turtle84375 Node.js is better for performance boosting, since asynchronous non-blocking architecture is important for network I/O so when you need to call the authenticaton API (either for sign-up, log-in, or just to perform a basic account action) your site will not pause for an output. If you try doing the same with Python, it will be more difficult.

Onto the point, node.js is a runtime environment for building fast and scalable server-side/api applications. Python is a dynamic and multipurpose programming language.

This means using Python would be difficult for large-scaled projects and authentication APIs like so and it's also slower since it is an interpreted language, which converts every single line of code to machine code in order to execute it.

TL;DR python is slower and not very stable for this type of authentication projects.

**Comment by Turtle84375** [comment]
Gonna merge this into the dev branch for testing.

**Comment by Turtle84375** [comment]
https://dev.turtle84375.me now pulls from `dev`
